mitodl/release-script
# Add command to show status across all repos

#### What are the relevant tickets?
Fixes #327 

#### What's this PR do?
Adds a command to output status information for all repositories

#### How should this be manually tested?
Run `python bot_local.py bots status` in the console and check that the output makes sense to you.

#### Screenshots
![Screenshot from 2021-08-31 13-02-59](https://user-images.githubusercontent.com/863262/131545540-78ce5513-5515-4d0f-be38-85b3cb90bfd1.png)
